blockchain/types/transaction.go

```diff
@@ -78,7 +78,7 @@ type Transaction struct {
 	markedUnexecutable int32
 
 	// lock for protecting AsMessageWithAccountKeyPicker().
```

Comment:
We should need more general explanation for this mutex.
```suggestion
	// lock for protecting fields in Transaction struct
```

Reply:
Thanks :)